sdk/python/nemo-fabric-runtime/src/nemo_fabric/types.py

```
@@ -1603,6 +1680,47 @@ def response(self) -> JSONValue | None:
+            ):
+                raise FabricConfigError(
+                    f"{field.replace('_', ' ')} must be a nonnegative integer"
+                )
+        cost = data.get("cost_usd")
+        if cost is not None and (
+            isinstance(cost, bool)
+            or not isinstance(cost, (int, float))
+            or not math.isfinite(cost)
+            or cost < 0
+        ):
+            raise FabricConfigError("cost usd must be a finite nonnegative number")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)